src/main/java/org/testeditor/fixture/web/WebDriverFixture.java

```diff
@@ -752,7 +810,34 @@ private void executeScript() {
             ((JavascriptExecutor) driver).executeScript(sb.toString());
         } catch (IOException e) {
             logger.error("Can't read java script", e);
-            throw new RuntimeException(e);
+            throw new FixtureException("could not execute javascript", //
+                    keyValues("executeScript", exeuteScript), e);
+        }
+    }
+
+    /**
+     * generate a map compatible to FixtureException calls by combining the even
+     * numbered portion of the var args into key, value pairs
+     * 
+     * @param objects
+     * @return
+     */
+    public static Map<String, Object> keyValues(Object... objects) {
```

Comment:
limit to `private` scope? Especially since this method is very easy to use wrongly by accident (odd number of arguments)

Reply:
moved it over to the FixtureException itself